authLdap.php

```diff
@@ -362,13 +360,17 @@ function authLdap_login($user, $username, $password, $already_md5 = false)
                 $user_info['user_email'] = $username . '@example.com';
             }
             $userid = wp_insert_user($user_info);
+            if (is_wp_error($userid)) {
+                authLdap_debug('Error creating user : ' . $userid->get_error_message());
+                trigger_error('Error creating user: ' . $userid->get_error_message());
+                return $userid;
+            }
```

Comment:
Because I removed the wp_update_user() call so it only made sense to check for an error after the wp_insert_user() call.  But given that it needs to be put back, it makes sense to put the error check back as well.